OpenRA.Mods.Common/Traits/Repairable.cs

```diff
@@ -142,15 +142,20 @@ public Actor FindRepairBuilding(Actor self)
 
 		static void TryCallTransport(Actor self, Target target, Activity nextActivity)
 		{
-			var transport = self.TraitOrDefault<ICallForTransport>();
```

Comment:
This can simplify down to something like:
```csharp
static void TryCallTransport(Actor self, Target target, Activity nextActivity)
{
	var delta = (self.CenterPosition - target.CenterPosition).LengthSquared;
	var transport = self.TraitsImplementing<ICallForTransport>()
		.FirstOrDefault(t => t.MinimumDistance.LengthSquared < delta);

	var targetCell = self.World.Map.CellContaining(target.CenterPosition);
	if (transport != null)
		transport.RequestTransport(self, targetCell, nextActivity);
}
```